examples/tests/testing/test_portal.py

```diff
@@ -0,0 +1,3 @@
+from examples.testing.portal import test_portal_example_event, test_portal_example_sleep
+
+__all__ = ["test_portal_example_sleep", "test_portal_example_event"]
```

Comment:
What is the purpose of this?

Reply:
@Goldziher Requested the examples get run during testing. Since the examples are valid tests, they can be imported and _pytest_ will run them as-is. I added the `__all__` because _pycln_ will nuke them as unused imports without it.

Reply:
How about renaming `examples/testing/portal.py` to `examples/testing/test_with_portal.py`? Then it should get automatically get collected by pytest.

Reply:
I like the idea about renaming. I agree _examples/tests/testing/test_portal.py_ was fairly hacky.

Reply:
This is done